src/tasks/AppleAppBuilder/Templates/CMakeLists-librarymode.txt.template

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(%ProjectName%)
+enable_language(OBJC ASM)
+
```

Comment:
I'm a bit confused why we still have this file if LibraryBuilder isn't using CMake anymore, is this just for our tests then? shouldn't that use the LibraryBuilder then too?

if we want to keep using CMake for our tests internally then I'd prefer instead of a separate template if we could use the existing one and just pass in some IS_LIBRARY_MODE bool since the two templates are 90% the same.

Reply:
This will be done in a follow up PR.